src/webui/www/private/views/rss.html

```diff
@@ -0,0 +1,815 @@
+<style>
+    #rssView {
+        padding: 20px 20px 0 20px;
+        height: calc(100% - 20px);
+    }
+
+    #rssConentView {
+        display: table;
+        width: 100%;
+        height: calc(100% - 30px);
+        vertical-align: top;
+    }
+
+    #rssFeedFixedHeaderDiv .dynamicTableHeader, #rssArticleFixedHeaderDiv .dynamicTableHeader {
+        cursor: default;
+    }
+
+    .alignRight {
+        float: right;
+    }
+
+    .unreadArticle {
+        color: blue;
+    }
+
+    #rssFetchingDisabled {
+        color: red;
+        font-style: italic;
+        margin-bottom: 10px;
+    }
+
+    #centerRssColumn {
+        margin: 0 10px 0 10px;
+    }
+
+    #leftRssColumn, #centerRssColumn, #rightRssColumn {
+        float: left;
+        /*should be 20 px but due to rounding differences some browsers don't render that properly*/
+        width: calc(calc(100% - 21px) / 3); 
+        border: none;
+    }
+
+    #rightRssColumn {
+        overflow: auto;
+    }
+
+    #rssFeedTableDiv, #rssArticleTableDiv {
+        height: calc(100vh - 180px);
+    }
+
+    #rssTorrentDetailsName {
+        background-color: #678db2;
+        padding: 0;
+        color: white;
+    }
+
+    #rssTorrentDetailsDate {
+        background-color: #EFEFEF;
+    }
+
+    #rssDetailsView {
+        height: calc(100vh - 135px);
+        overflow: auto;
+    }
+</style>
+
+<div id="rssView">
+    <div id="rssTopBar">
+        <div id="rssFetchingDisabled" class="invisible">
+            QBT_TR(Fetching of RSS feeds is disabled now! You can enable it in application settings.)QBT_TR[CONTEXT=RSSWidget]
+        </div>
+        <div id="rssButtonBar" style="overflow: hidden; height: 30px;">
```

Comment:
I'd prefer if you replaced most of this inline styling with rules defined in the `<style>` element above

Reply:
Changed things for `rss.html` and `rssDownloader.html`. I think for the dialog boxes the inline styling is okay.